scripts/dev.sh

```diff
@@ -0,0 +1,539 @@
+     apply_og
+     apply_subscribe
+  elif [[ "$COMMAND" == "init" ]]; then
+     init_cluster
+     login_registry
+  elif [[ "$COMMAND" == "build" ]]; then
+     build
+     bundle
+     catalog
+  elif [[ "$COMMAND" == "catalog" ]]; then
+     apply_catalog
+  elif [[ "$COMMAND" == "subscribe" ]]; then
+     apply_og
+     apply_subscribe
+  elif [[ "$COMMAND" == "login" ]]; then
+     login_registry
+  elif [[ "$COMMAND" == "e2e" ]]; then
+     add_rbac
+     install_rook
+     install_serverless
+     setup_knative_serving
+     install_cert_manager
+     add_affinity_label_to_node
+     create_image_content_source_policy
+  elif [[ "$COMMAND" == "scorecard" ]]; then
+     run_scorecard
+  elif [[ "$COMMAND" == "deploy" ]]; then
+     deploy
+  else 
+    echo
+    echo "Command $COMMAND unrecognized."
+    echo
+    echo "${USAGE}"
+    exit 1
+  fi
+
+}
+
+#############################################################################
+# Setup an OCP cluster to use the private registry, insecurely (testing only)
+#############################################################################
+init_cluster() {
+    oc patch configs.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge
+    OCP_REGISTRY_URL=${OCP_REGISTRY_URL:=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')}
+    oc patch image.config.openshift.io/cluster  --patch '{"spec":{"registrySources":{"insecureRegistries":["'$OCP_REGISTRY_URL'"]}}}' --type=merge
+}
+
+login_registry() {
+    echo  $CONTAINER_COMMAND login $TLS_VERIFY -u kubeadmin -p $(oc whoami -t) $OCP_REGISTRY_URL
+    $CONTAINER_COMMAND login $TLS_VERIFY -u kubeadmin -p $(oc whoami -t) $OCP_REGISTRY_URL
+    oc registry login --skip-check   
+}
+
+apply_catalog() {
+    CATALOG_FILE=/$TEMP_DIR/catalog.yaml    
+    
+cat << EOF > $CATALOG_FILE
+    apiVersion: operators.coreos.com/v1alpha1
+    kind: CatalogSource
+    metadata:
+      name: open-liberty-operator-catalog
+      namespace: $NAMESPACE
+    spec:
+      sourceType: grpc
+      image: $CATALOG_IMG
+      imagePullPolicy: Always
+      displayName: Open Liberty Catalog
+      updateStrategy:
+        registryPoll:
+          interval: 1m
+EOF
+
+    oc apply -f $CATALOG_FILE
+    rm  $CATALOG_FILE
+}
+
+apply_subscribe() {
+    SUBCRIPTION_FILE=/$TEMP_DIR/subscription.yaml    
+    
+cat << EOF > $SUBCRIPTION_FILE
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: open-liberty-operator-subscription
+  namespace: $NAMESPACE
+spec:
+  channel:  beta2
+  name: open-liberty
+  source: open-liberty-operator-catalog
+  sourceNamespace: $NAMESPACE
+  installPlanApproval: Automatic
+EOF
+
+    oc apply -f $SUBCRIPTION_FILE
+    rm $SUBCRIPTION_FILE          
+}
+
+apply_og() {
+    OG_FILE=/$TEMP_DIR/og.yaml    
+    
+cat << EOF > $OG_FILE
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: rco-operator-group
```

Comment:
Should be "open-liberty-operator-group" rather than rco?

Reply:
Yes. I kept it consistent and went with ol-operator-group.